src/server/create/controllers/start.js

```diff
@@ -1,8 +1,13 @@
 import { sessionNames } from '~/src/server/common/constants/session-names'
 import { saveToCreate } from '~/src/server/create/helpers/form'
+import { isCreateFeatureTemporaryDisabled } from '~/src/server/create/helpers/feature-toggle/create-feature-flag'
 
 const startController = {
   handler: async (request, h) => {
+    if (isCreateFeatureTemporaryDisabled(request)) {
+      return h.redirect('/create/disabled')
```

Comment:
might be able to just render the holding page directly here rather than redirecting to a different controller.

Reply:
Yeah could that , less abstaction